omod/src/main/webapp/resources/styles/bootstrap-editable.css

```diff
@@ -661,3 +661,35 @@ a.editable-click.editable-disabled:hover {
     margin-left: -5px;
     margin-right: -5px;
 }
+ 
```

Comment:
bootstrap-editable is a standard css downloaded from the internet, if we are adding new css, we should use our files